precompile/allow_list.go

```diff
@@ -43,12 +43,17 @@ var (
 
 // AllowListConfig specifies the initial set of allow list admins.
 type AllowListConfig struct {
-	AllowListAdmins []common.Address `json:"adminAddresses"`
+	AllowListAdmins  []common.Address `json:"adminAddresses"`
+	EnabledAddresses []common.Address `json:"enabledAddresses,omitempty"` // initial enabled addresses
 }
 
 // Configure initializes the address space of [precompileAddr] by initializing the role of each of
 // the addresses in [AllowListAdmins].
 func (c *AllowListConfig) Configure(state StateDB, precompileAddr common.Address) {
+	// First set enabled roles so these can be upgraded to admin addresses below.
```

Comment:
Hmm good call, should we be more strict here and require that there are no overlaps between the two lists?

Reply:
added verify to allowlist

Reply:
No that we added this within `Verify()` can we remove this comment?

Reply:
Can we also add unit tests for verify within the `precompile` package?

Reply:
done